packages/cms-cli/lib/prompts.js

```diff
@@ -1,7 +1,4 @@
-const API_KEY_REGEX = new RegExp(
-  '^([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})$',
-  'i'
-);
+const API_KEY_REGEX = /^([a-z0-9]{8}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{4}-[a-z0-9]{12})$/gi;
```

Comment:
The global match isn't needed since we're also capturing beginning/end of string.